javascript-timers/interval.js

```diff
@@ -0,0 +1,12 @@
+var h1 = document.querySelector('h1');
+
+function countdown() {
+  var newH1 = h1.textContent - 1;
```

Comment:
Instead of leaving the "truth" in the DOM, use a variable to store the current countdown number.

Reply:
Makes sense! Just pushed the new file.